canonical-web-and-design/ubuntu.com
# Improve the small screen view of the subscriptions

## Done
Provide an entirely new UI for small screens.

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Login
- Shrink the screen to a small screen or better yet use your phone
- Is there that information is accessible

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8394
